File: p009.py
#!/usr/bin/python
for a in range(1, 500):
    for b in range(1, 500):
        for c in range(1, 500):
            if(a*a+b*b==c*c):
                if(a+b+c==1000):
                    print a*b*c
                    break

File: p009.pl
#!/usr/bin/perl -w

for ($a = 1; $a <= 500; $a++){
  for ($b = $a+1; $b <= 500; $b++){
    for ($c = $a+2; $c <=500; $c++){
      if ($c == sqrt($a*$a+$b*$b)){
        if (1000 == $a + $b + $c){
	  print($a * $b * $c, "\n");
	  exit;
	}
      }
    }
  }
}

